Tasks/VsTest/task.json

```diff
@@ -1,171 +1,178 @@
-    }
-  ],
-  "instanceNameFormat": "Test Assemblies $(testAssembly)",
-  "execution": {
-    "PowerShell": {
-      "target": "$(currentDirectory)\\VSTest.ps1",
-      "argumentFormat": "",
-      "workingDirectory": "$(currentDirectory)"
+        },
+        {
+            "name": "pathtoCustomTestAdapters",
+            "type": "string",
+            "label": "Path to Custom Test Adapters",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Directory path to custom test adapters. Nuget restored adapters are automatically searched for.",
+            "groupName": "advancedExecutionOptions"
+        },
+        {
+            "name": "otherConsoleOptions",
+            "type": "string",
+            "label": "Other console options",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Other Console options that can be passed to vstest.console.exe. Click on the help link below for more details.",
+            "groupName": "advancedExecutionOptions"
+        },
+        {
+            "name": "testRunTitle",
+            "type": "string",
+            "label": "Test Run Title",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Provide a name for the Test Run.",
+            "groupName": "reportingOptions"
+        },
+        {
+            "name": "platform",
+            "type": "string",
+            "label": "Platform",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Platform against which the tests should be reported. If you have defined a variable for platform in your build task, use that here.",
+            "groupName": "reportingOptions"
+        },
+        {
+            "name": "configuration",
+            "type": "string",
+            "label": "Configuration",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Configuration against which the tests should be reported. If you have defined a variable for configuration in your build task, use that here.",
+            "groupName": "reportingOptions"
+        },
+        {
+            "name": "publishRunAttachments",
+            "type": "boolean",
+            "label": "Upload Test Attachments",
+            "defaultValue": "true",
+            "required": false,
+            "helpMarkDown": "Opt in/out of publishing test run level attachments.",
+            "groupName": "reportingOptions"
+        }
+    ],
+    "instanceNameFormat": "Test Assemblies $(testAssembly)",
+    "execution": {
+        "PowerShell": {
+            "target": "$(currentDirectory)\\VSTest.ps1",
+            "argumentFormat": "",
+            "workingDirectory": "$(currentDirectory)",
+            "platforms": [
+                "windows"
+            ]
+        },
+        "Node": {
+            "target": "VSTest.js",
+            "argumentFormat": ""
```

Comment:
I don't believe node handler uses argumentFormat.  Not that it's bad - just ignored